setup.py

```diff
@@ -5,12 +5,13 @@
 
 setuptools.setup(
     name='caper',
-    version='1.0.0',
+    version='1.0.1',
     python_requires='>=3.6',
     scripts=[
         'bin/caper',
         'bin/run_mysql_server_docker.sh',
         'bin/run_mysql_server_singularity.sh',
+        'scripts/gcp_caper_server/create_instance.sh',
```

Comment:
Seems to conflict with documentation, which instructs users to invoke with `./create_instance.sh`, which won't work if the use caper installed script.

Reply:
Oh, that's a good point. This script doesn't have to be run on the directory where it resides. I will change it to `bash create_instance.sh`.